source/developers/minio-drivers.rst

```diff
@@ -18,43 +18,40 @@ MinIO publishes the following Software Development Kits (SDK):
 4. :ref:`Java <java-sdk>`
 5. :ref:`JavaScript <javascript-sdk>`
 6. :ref:`Python <python-sdk>`
+7. :ref:`C++ <cpp-sdk>`
 
+.. _cpp-sdk:
 
-..
-  C++ repo does not have any releases yet. Once released, unblock this section and add to toctree and numbered list.
-  
-  .. _cpp-sdk:
+C++ (``minio-cpp``)
+-------------------
 
-  C++ (``minio-cpp``)
-  -------------------
+Latest Version
+  |cpp-sdk-version|
```

Comment:
Hm. Dang. This requires the Makefile to sync the latest version.

Also https://github.com/minio/minio-cpp implies there has not been a relase in an entire year. I am not sure that is correct (@balamurugana ? )

Removing this for now until we have an answer.